src/console_cmds.cpp

```diff
@@ -614,6 +615,115 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	IniFile ini { group_names };
+	ini.LoadFromDisk(path, NO_DIRECTORY);
+	const IniGroup *ban_group = ini.GetGroup("bans", false);
+
+	if (ban_group == nullptr) return -1;
+
+	int loaded_count = 0;
+	for (const IniItem *item = ban_group->item; item != nullptr; item = item->next) {
+		// banned clients are added to global ban list when kicked or banned
+		NetworkServerKickOrBanIP(item->name.c_str(), true, nullptr);
+		loaded_count++;
+	}
+	return loaded_count;
+}
+
+/**
+* Loads a ban list from a user-specified file. The entries in the list are added to the current collection of banned entries.
+* @see ConBanList
+*/
+DEF_CONSOLE_CMD(ConBanListLoad)
+{
+	if (argc == 0) {
+		IConsoleHelp("Loads the IP's of banned clients from a file: Usage 'banlist_load <path_to_file>'");
+		return true;
+	}
+
+	if (argc != 2) {
+		return false;
+	}
+
+	const char * const path = argv[1];
+	IConsolePrintF(CC_INFO, "Loading banlist from '%s'...", path);
+	int loaded_entries_count = LoadBanList(path);
+	if (loaded_entries_count == -1) {
+		IConsolePrint(CC_WARNING, "File does not exist or [bans] section is unspecified.");
+		return false;
+	}
+
+	IConsolePrintF(CC_INFO, "Loaded %d entries.", loaded_entries_count);
+	return true;
+}
+
+/**
+* Saves the specified ban list to the specified path.
+* @param path The path to save the ban list to.
+* @param ban_list The ban list to save.
+* @return The number of entries saved to the file or -1 if an error occurred while saving entries.
+*/
+static int SaveBanList(const std::string &path, const StringList &ban_list)
+{
+	const char * const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	IniFile ini { group_names };
+	IniGroup *ban_group = ini.GetGroup("bans", true);
+
+	int item_count = 0;
+	for (const std::string &name : ban_list) {
+		ban_group->GetItem(name, true);
+		item_count++;
+	}
+
+	if (ini.SaveToDisk(path)) {
+		return item_count;
+	} else {
+		return -1;
+	}
+}
+
+/**
+* Saves the current ban list to a user-specified file.
+* @see ConBanList
+*/
+DEF_CONSOLE_CMD(ConBanListSave)
+{
+	if (argc == 0) {
+		IConsoleHelp("Saves the banned list of clients to a file: Usage 'banlist_save <path_to_file>'");
```

Comment:
The deviates from the style used for the other help messages, and I think it should be IPs (plural) not IP's (possessive).
`Save the IPs of banned clients to a file. Usage 'banlist_save <path_to_file>`